contracts/MarginBase.sol

```diff
@@ -200,6 +202,12 @@ contract MarginBase is MinimalProxyable {
         emit Withdraw(msg.sender, _amount);
     }
 
+    /*///////////////////////////////////////////////////////////////
+                    Account Deposit & Open Position
```

Comment:
Wondering if we should be doing these section type labels for singular functions like this. Might make more sense to break down the section labels by product moving forward. ie. Account Management, Cross Margin, Advanced Orders

Reply:
Yeah, it is debatable. I really just followed suit from rari-capital's contracts. I liked the organization by functionality vs the typical way.